src/MainView.vala

```diff
@@ -148,7 +148,7 @@ public class OnlineAccounts.MainView : Gtk.Grid {
         remove_button.clicked.connect (() => {
             var message_dialog = new Granite.MessageDialog (
                 _("Remove “%s” from this device").printf (e_source.display_name),
-                _("This account will be removed and will no longer appear in apps on this device."),
+                _("This account will be removed and no longer appear in “Apps” on this device."),
```

Comment:
I'm not sure why you added quotation marks here. The account will be removed from any apps it would normally show in. Here, "apps" refers to an unknown number of applications, not a location.

Reply:
I thought it was that top left menu with all the apps.

Reply:
```suggestion
                _("This account will be removed and no longer appear in any apps on this device."),
```